### DISCUSSION TITLE
Creating aliases (refactoring?)

### DISCUSSION THREAD

**Discussion by Awethon** [Flow-Launcher/Flow.Launcher#2764] [Ideas]
To create aliases custom query shortcuts feature was introduced. However it doesn't provide smooth UX in my opinion.
My use case is that for alternative/3rd party apps I want to be able to find an app by typing the name of the original app and autocomplete and suggestions to behave like I'm searching for the original app. Currently, unless I type a full name, I won't see any suggestions.

I need it for Unigram -> Telegram, but out of my head use cases can also be vpn -> (name of vpn service), word -> libreoffice.

**Comment by onesounds** [comment]
It's not impossible, but I think it would make the structure too complicated.
How about create another shortcut? (.lnk) 

<img src="https://github.com/Flow-Launcher/Flow.Launcher/assets/6903107/ba0585a1-3b53-48b8-b576-32e5e3f32577" width="200">

I think you can do that by creating another shortcut for "libreoffice" and renaming it to “word”. There can be two lnks in an executable, so use whichever you need. I actually use it like this. (In this case, you can also modify the icon at will.)

**Reply by Awethon** [reply]
Unfortunately this is not easy to do for Microsoft Store apps
